test/cypress/integration/simplified-account/service-settings/stripe-details/confirm-org-details.cy.js

```diff
@@ -0,0 +1,307 @@
+
+          cy.get('.govuk-summary-list__row')
+            .last()
+            .contains('McDuck Manor').should('exist')
+            .contains('Duckburg').should('exist')
+            .contains('SW1A 1AA').should('exist')
+
+          cy.get('input[type="radio"]')
+            .siblings('label')
+            .contains('Yes, these organisation details match')
+            .prev('input[type="radio"]')
+            .check()
+
+          cy.get('#confirm-org-details-form button[type="submit"]').click()
+          cy.title().should('eq', 'Settings - Stripe details - GOV.UK Pay')
+          cy.get('h1').should('contain', 'Stripe details')
+          cy.location('pathname').should('not.contain', '/organisation-details/index')
+          cy.get('.govuk-task-list__item')
+            .contains('Confirm your organisation\'s name and address match your government entity document')
+            .parent()
+            .parent()
+            .within(() => {
+              cy.get('.govuk-task-list__status').should('contain.text', 'Complete')
+            })
+        })
+      })
+      describe('When selecting no', () => {
+        describe('When submitting valid organisation details', () => {
+          beforeEach(() => {
+            setStubs({}, [
+              ...Array(3).fill(stripeAccountSetupStubs.getStripeSetupProgressByServiceExternalIdAndAccountType({
+                serviceExternalId: SERVICE_EXTERNAL_ID,
+                accountType: LIVE_ACCOUNT_TYPE
+              })),
+              gatewayAccountStubs.getStripeAccountByServiceIdAndAccountType(
+                SERVICE_EXTERNAL_ID,
+                LIVE_ACCOUNT_TYPE,
+                {
+                  stripeAccountId: STRIPE_ACCOUNT_ID
+                }
+              ),
+              stripePspStubs.updateAccount({
+                stripeAccountId: STRIPE_ACCOUNT_ID
+              }),
+              stripeAccountSetupStubs.patchStripeProgressByServiceExternalIdAndAccountType({
+                serviceExternalId: SERVICE_EXTERNAL_ID,
+                accountType: LIVE_ACCOUNT_TYPE
+              }),
+              serviceStubs.patchUpdateServiceGatewayAccounts({
+                serviceExternalId: SERVICE_EXTERNAL_ID
+              }),
+              stripeAccountSetupStubs.getStripeSetupProgressByServiceExternalIdAndAccountType({
+                serviceExternalId: SERVICE_EXTERNAL_ID,
+                accountType: LIVE_ACCOUNT_TYPE,
+                organisationDetails: true
+              })
+            ])
+            cy.visit(STRIPE_DETAILS_SETTINGS_URL + '/organisation-details/index')
+          })
+
+          it('should redirect to the task summary page on success', () => {
+            cy.get('input[type="radio"]')
+              .siblings('label')
+              .contains('No, these organisation details do not match')
+              .prev('input[type="radio"]')
+              .check()
+
+            cy.get('#confirm-org-details-form button[type="submit"]').click()
```

Comment:
How is this line differentiated from [this same one above for a 'yes' selection](https://github.com/alphagov/pay-selfservice/pull/4378/files#diff-666005f7faf40ee7f1e88758b19501f4f7d86ea256202d2f5817e7b83d7ff309R169)?

Reply:
the difference being that on one test the radio 'yes' is checked before hitting the submit button and on this test the radio 'no' is checked? I'm not sure I follow.

Reply:
Ohhh sorry lol